numpy/core/code_generators/generate_umath.py

```diff
@@ -360,15 +361,15 @@ def english_upper(s):
           docstrings.get('numpy.core.umath._ones_like'),
           'PyUFunc_OnesLikeTypeResolver',
           TD(noobj),
-          TD(O, f='Py_get_one'),
+          TD(P),
           ),
 'power':
     Ufunc(2, 1, None,
           docstrings.get('numpy.core.umath.power'),
           None,
           TD(ints),
           TD(inexact, f='pow', astype={'e':'f'}),
-          TD(O, f='npy_ObjectPower'),
+          TD(O, f='PyNumber_Power'),
```

Comment:
This doesn't look safe to me - we don't pass enough arguments to `PyNumber_Power` in the loop - `npy_ObjectPower` is there to add an extra `Py_None` argument